### PR TITLE
initial file for sanford bio

### PR DESCRIPTION
**Pull request by shmitr**
I was told to do this, so I did. It is the full story of my life.  Nothing is omitted. #17

### CODE REVIEW

File: bios/sanford-bio.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+##  Sanford's Bio
+
+- list items
+
+- [ ] not sure what a checkbox does here.
```
Comment (shmitr) [inline bios/sanford-bio.md:5]: Then why is the checkbox here.  Maybe you should delete it...

**Comment by shmitr** [comment]
:-1: It's just an example dude...

File: bios/sanford-bio.md (hunk 1 above)
Comment (ernie-m) [inline bios/sanford-bio.md:5]: sanford, your bio is whack!  :)

**Comment by ernie-m** [comment]
don't make excuses sanford!

**Comment by shmitr** [comment]
:shipit: ok - here we go!

**Pull request merged**